### PR TITLE
Feature/bossa source

### PR DESCRIPTION
**Pull request by mkopec87**
New source: https://bossa.pl/ Polish broker.

### CODE REVIEW

**Comment by mkopec87** [comment]
@adri0 , does it work for you? Cannot reproduce this locally :(

**Comment by adri0** [comment]
Nope. Good news is that I re-ran the job and it worked :D

File: investd/sources/bossa.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+from ..model import Action, AssetType, Currency, Transaction
+from .base import SourceBase
+
+COL_AMOUNT = "wartość"
+COL_CURRENCY = "waluta"
+COL_DATE = "data"
+COL_PAPER = "papier"
+COL_PRICE = "cena"
+COL_QUANTITY = "ilość"
```
Comment (adri0) [inline investd/sources/bossa.py:14]: It's weird that Github doesn't automatically detect UTF8 encoding. :(

**Review by adri0 — APPROVED** [review]

File: investd/sources/bossa.py (hunk 1 above)
Comment (mkopec87) [reply investd/sources/bossa.py:14]: it may not be utf8 actually, since the file I parse is in windows-1250

**Pull request merged**